src/components/views/settings/index.tsx

```diff
@@ -163,7 +178,21 @@ export const Settings: React.FC<SettingsProps> = ({navigation}) => {
                   accessibilityLabel={label}
                   accessibilityRole="button"
                   accessibilityHint={hint}
-                  onPress={() => navigation.navigate(screen)}>
+                  onPress={() =>
+                    screen === 'test'
+                      ? Linking.openURL(
+                          `mailto:support@example.com?subject=App Feedback for COVID Alert NY: Version ${version}&body=\n\n\n\n
+                          Device: ${getModel()}
+                          OS version: ${Platform.Version}
+                          Status of EN framework: ${exposure.enabled}
+                          Bluetooth active: ${
+                            type.length > 0
+                              ? type.indexOf(StatusType.bluetooth) === -1
+                              : 'active'
```

Comment:
This can print `true`, `false` or `'active'`. According to the issue it should be `'yes'` or `'no'`. We definitely shouldn't have `true` and `'active'` signifying different things.

Reply:
No need for this if we do the above. We can't detect ourselves the BT status we're realying on ENS to provide hat and all the ens info includes that.